dbee/adapters/postgres_driver.go

```diff
@@ -56,6 +56,7 @@ func (c *postgresDriver) Structure() ([]*core.Structure, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer rows.Close()
```

Comment:
I'm a bit new to go, I'm not sure if it's best practice to do it like this (err return), or if to put something like this before the err return:

```go
if rows != nil {
	defer rows.Close()
}
```